Resources/doc/operations.md

```diff
@@ -51,9 +30,7 @@ with an empty array as first parameter:
     resource.product:
         parent:    "api.resource"
         arguments: [ "AppBundle\Entity\Product" ]
-        calls:
-            -      [ "initItemOperations", [ [ ] ] ]
-            -      [ "initCollectionOperations", [ [ "@resource.product.collection_operation.get" ] ] ]
+        properties: properties: { collectionOperations: ["GET"], itemOperations: ["GET", "PUT"] }
```

Comment:
Looks like you've a typo with two `properties` right ? Moreover, can you use a clearer syntax in multi-lines ?